src/figuresClasses.ts

```diff
@@ -1,19 +1,191 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
 export class Triangle {
+  shape: 'triangle';
+
+  color: 'red' | 'green' | 'blue';
```

Comment:
You can remove redundant spaces 
```suggestion
  shape: 'triangle';
  color: 'red' | 'green' | 'blue';
```

Reply:
![image](https://github.com/mate-academy/ts_figures_classes/assets/51528776/09e508fd-d2e5-47e1-9b76-6b8dd315fe66)